src/app/(nextjs_migration)/(guest)/page.tsx

```diff
@@ -14,10 +14,25 @@ import MailImage from "../../../client/images/landing-mail@2x.webp";
 import NaturePhoneImage from "../../../client/images/landing-nature-phone@2x.webp";
 import { getNonce } from "../functions/server/getNonce";
 import { PageLoadEvent } from "../components/client/PageLoadEvent";
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../api/utils/auth";
+import { isFlagEnabled } from "../../functions/server/featureFlags";
 
-export default function Home() {
+export default async function Home() {
   const l10n = getL10n();
 
+  const session = await getServerSession(authOptions);
+  const userId = session?.user?.subscriber?.id; // FIXME what if the user is not logged in? Need a "device ID" or equivalent (cookie, etc)
+
+  // @see https://github.com/mozilla/experimenter/tree/main/cirrus
+  const serverUrl = process.env.SERVER_URL ?? "http://localhost:6060";
```

Comment:
Q: Should this be `SERVER_URL` or `NEXT_PUBLIC_SERVER_URL` since it looks like client front end code?

Reply:
This is a server component actually - it needs to be in order to communicate with Cirrus. It then passes userId as a prop to the client component (where it's used by Glean).

Reply:
(I'm actually not totally sure which URL we need to use here to talk to the Cirrus sidecar... I think it might need to be hardcoded to `localhost` with a particular port)

Reply:
OK I've moved this to a shared function, and I think I've figured out how to call the Cirrus sidecar.